### ISSUE TITLE
Rebase goconst

### ISSUE THREAD

**Issue by SuperQ** [golangci/golangci-lint#1009]
Thank you for creating the issue!

- [x] Yes, I'm using a binary release within 2 latest major releases. Only such installations are supported.
- [x] Yes, I've searched similar issues on GitHub and didn't find any.
- [x] Yes, I've included all information below (version, config, etc).

Please include the following information:

```console
$ golangci-lint --version
golangci-lint has version 1.24.0 built from 6fd4383 on 2020-03-15T11:38:02Z
```

It would be good to rebase https://github.com/golangci/goconst to pick up the commit in the upstream master that [matches function params](https://github.com/jgautheron/goconst/commit/cda7ea3bf591db274392c3619cd82f83bb4f6fbb).

**Comment by gonzaloserrano** [comment]
Also interested in this.

**Comment by SVilgelm** [comment]
It would be better to get rid of the golangci/goconst repo and switch to the main goconst, but it requires to move some code from golangci/goconst to the golangci-lint repo

**Comment by gonzaloserrano** [comment]
Sorry, I'm kind of new to golangci-lint internals. Since `golangci/goconst` has diverged from the original goconst (it's 2 commits ahead and 1 behind it), what's needed here? Rebase `golangci/goconst` and move the code to this repo?

**Comment by SVilgelm** [comment]
So, the `golangci/goconst` contains some code that add a support of `goconst` (`Run` function). I think it's better to move the code that has been added to the `golangci-lint` repo and have a direct import of `goconst` module. So we will be able to update the `goconst` if needed.

**Comment by ldez** [comment]
Closed by #1500
